main.py

```diff
@@ -0,0 +1,36 @@
+from kvstore.constants import GET_OP, SET_OP, NULL
+from kvstore.input import get_input
+
+
+class KVStore:
+    def __init__(self):
+        self.store = {}
+
+    def get(self, key):
+        return self.store.get(key, NULL)
+
+    def set(self, key, value):
+        self.store[key] = value
+        return value
+
+
+def main():
+    kvstore = KVStore()
+
+    while True:
+        try:
```

Comment:
You only expect get_input to throw, so you may want to make the scope of your try/except to just this function